### DISCUSSION TITLE
Compatibility Issue Between Blowfish Theme and Latest Hugo Version

### DISCUSSION THREAD

**Discussion by Yajie-Xu** [nunocoracao/blowfish#1882] [Q&A]
Hi everyone,

I recently noticed that Hugo has been updated, and since then, my website deployment on GitHub Pages has started failing. I’m planning to rebuild my website using the Blowfish theme. However, when I set up a new site using the command `npx blowfish-tools` as suggested in the documentation, I encountered the following warning after running `hugo server`:

`WARN Module "blowfish" is not compatible with this Hugo version: 0.87.0/0.139.3; run "hugo mod graph" for more information.`

I’m concerned that this incompatibility might also cause deployment failures with the new site. Could anyone provide advice? I’d appreciate your guidance.

**Comment by atasoglou** [comment]
Getting the same... Maybe related to #1635?
